fictadvisor-api/src/v2/api/controllers/PageTextController.ts

```diff
@@ -27,16 +24,9 @@ export class PageTextController {
     private pageTextService: PageTextService,
   ) {}
 
-  @ApiOkResponse({
-    type: [PageTextsResponse],
-  })
-  @ApiBadRequestResponse({
-    description: `\n
-    InvalidQueryException:
-      Keys can not be empty`,
-  })
   @ApiEndpoint({
     summary: 'Get all texts for page',
+    documentation: PageTextDocumentation.GET_ALL_TEXTS,
```

Comment:
Name of the documentation property (GET_ALL_TEXTS) must be the same as corresponding controller method (GET_ALL)